lib/elements/element.py

```diff
@@ -150,16 +150,26 @@ def set_param(self, name, value):
     @property
     @cache
     def style(self):
```

Comment:
The only thing that actually uses this property is `has_style()` and `get_style()`.  Maybe we can get rid of this and just use `parse_style()` in `get_style()`?